lib/ClusterShell/Worker/Ssh.py

```diff
@@ -58,7 +58,12 @@ def _build_cmd(self):
         options = task.info("ssh_options")
 
         # Build ssh command
-        cmd_l = [ path, "-a", "-x"  ]
+        cmd_l = path.split()
```

Comment:
Use shlex.split(path) - indeed shlex handles whitespaces and quotes, we should had used it before!